pack.go

```diff
@@ -93,7 +93,9 @@ type PackManifestOptions struct {
 	// Layers is the layers of the manifest.
 	Layers []ocispec.Descriptor
 
-	// ManifestAnnotations is the annotation map of the manifest.
+	// ManifestAnnotations is the annotation map of the manifest. Set
+	// "org.opencontainers.image.created" with a value to make PackManifest
```

Comment:
Developers is not encouraged to use string literals but constants. Please refer the developers to use constants from ocispec.

Reply:
This comment is a duplicate of https://github.com/oras-project/oras-go/pull/749/files#r1575970704

Reply:
Actually we are talking about different stuffs😀 In [another comment](https://github.com/oras-project/oras-go/pull/749#discussion_r1575970704) my concern was that users may not understand what `AnnotationArtifactCreated` and `AnnotationCreated` are as they are not defined in `oras-go`, and we may want to use string literals in the comment.

https://github.com/oras-project/oras-go/blob/6fce64e956e850a3af7ea16e85959c23e3eddc4b/pack.go#L50-L54

Reply:
> Actually we are talking about different stuffs😀 In [another comment](https://github.com/oras-project/oras-go/pull/749#discussion_r1575970704) my concern was that users may not understand what `AnnotationArtifactCreated` and `AnnotationCreated` are as they are not defined in `oras-go`, and we may want to use string literals in the comment.
> 
> https://github.com/oras-project/oras-go/blob/6fce64e956e850a3af7ea16e85959c23e3eddc4b/pack.go#L50-L54

Has a point. Kept using the string literal in the comment for the error value, and replaced with the variable name `ocispec.AnnotationCreated` for comments of the function and annotation map.